packages/services/src/infrastructure/api/hiro/hiro-stacks-api.client.ts

```diff
@@ -1,38 +1,145 @@
-import type { Transaction } from '@stacks/stacks-blockchain-api-types';
+import { FtMetadataResponse } from '@hirosystems/token-metadata-api-client';
+import {
+  AddressBalanceResponse,
+  AddressTransactionsListResponse,
+  MempoolTransactionListResponse,
```

Comment:
Alternative to redefining below


```suggestion
  AddressBalanceResponse as HiroAddressBalanceResponse,
  AddressTransactionsListResponse as HiroAddressBalanceResponse,
  MempoolTransactionListResponse as MempoolTransactionListResponse,
```

Reply:
or even `import * as Hiro ....`?